shared/common-adapters/tab-bar.native.js

```diff
@@ -15,21 +16,80 @@ class TabBarItem extends Component {
   }
 }
 
+type SimpleTabBarButtonProps = {
+  selected: boolean,
+  label: string,
+  tabWidth?: ?number,
+  style?: Object
+}
+
+class SimpleTabBarButton extends Component<void, SimpleTabBarButtonProps, void> {
+  render () {
+    const tabWidth = this.props.tabWidth || 93
+    return (
+      <Box style={{...stylesTab, width: tabWidth}}>
+        <Text type='BodySemibold' style={{...stylesLabel, color: this.props.selected ? globalColors.black75 : globalColors.black60}}>
+          {this.props.label.toUpperCase()}
+        </Text>
+        {this.props.selected && <Box style={stylesSelectedUnderline}/>}
```

Comment:
I don't understand?

This was inline before in the original tab bar implementation and I moved it out into its own component